mne_connectivity/spectral/epochs_multivariate.py

```diff
@@ -103,6 +105,7 @@ class _EpochMeanMultivariateConEstBase(_AbstractConEstBase):
 
     n_steps = None
     patterns = None
+    con_scores_dtype = np.float64
```

Comment:
Do we need float64, or would float32 be sufficient?

Just can affect the RAM usage and storage.

Reply:
I just set to `float64` because that is the existing default for the other methods.

Just to clarify, the `float64` would be used for `mic`, `mim`, and `gc`. In the end, the memory usage for one connection of these methods wouldn't be any higher than for one connection of `coh` or `imcoh`.

For `cacoh`, the dtype used is complex128, so the memory usage would be identical to that of `cohy`.

Reply:
Ah okay sure that makes sense.

In case it comes up, future support of allowing float32 by the end-user might be useful for speeding up computation as well as improving storage and usage of RAM, but that's a future discussion. Would need to abstract the types out carefully.

Reply:
For sure, this was also mentioned to me as a feature request by someone. If others think it's worth pursuing I would be happy to look into it later on!

Reply:
This was the feature request in question: #168